doc/INSTALL.md

```diff
@@ -206,3 +207,123 @@ Alternatively, the kpatch and kpatch-build scripts can be run directly from the
 git tree.
 
 
+### OpenEuler
+
+*NOTE: You'll need about 15GB of free disk space for the kpatch-build cache in
+`~/.kpatch` and for ccache.*
+
+Install the dependencies for compiling kpatch and running kpatch-build:
+
+```bash
+source test/integration/lib.sh
+# Will request root privileges
+kpatch_dependencies
+```
+
+Before running kpatch-build, two more things need to be checked:
+-------
+1. Ensure current kernel compiled with *CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY* set
+
+    openEuler has two strategies to apply kernel live patches and it is decided at compile time.
+
+    When CONFIG_LIVEPATCH_STOP_MACHINE_CONSISTENCY set, openEuler uses its own strategy.
+
+    When CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY set, openEuler uses the conventional strategy.
+
+    Only one config option can take effect at the same time.
+    Differences between two strategies will not be discussed here, but you can easily guess them from config names.
```

Comment:
Does (Open)Euler document the differences anywhere?  If so, it would be nice if we could provide the user a link.

Reply:
After some searching work, I find their have their own version of kpatch which is https://gitee.com/src-openeuler/kpatch. I also find a blog written by the openEuler official which describes the differences. The linke is https://www.modb.pro/db/232858. Sadly, they are both written in chinese. I have add these information to the document.